source/_integrations/downloader.markdown

```diff
@@ -60,3 +60,7 @@ Along with the event the following payload parameters are available:
       message: "{{trigger.event.data.filename}} download failed"
       title: "Download Failed"
  ```
+
```

Comment:
the file path is discussed in line 19. Please move your comment there.

Reply:
Moved as suggested.